### PR TITLE
modified .ico file, button in form section fixed to appear bigger

### PR DESCRIPTION
**Pull request by KamilaKisiel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
     <title>Części Samochodowe - wyszukiwarka codeVIN</title>
     <link rel="stylesheet" type="text/css" href="reset.css">
     <link rel="stylesheet" type="text/css" href="style.css">
-    <link rel="icon" type="image/png" sizes="16x16" href="favicon.ico/favicon-16x16.png">
+    <link rel="shortcut icon" href="favicon.ico/logo_pic.ico" type="image/x-icon">
+    <!--<link rel="icon" type="image/png" sizes="32x32" href="favicon.ico/favicon-16x16.png">-->
```
Comment (RadekRo) [inline index.html]: What is a purpose of this comment? Can you explain it to me as a 6 y.o.? :)

**Review by RadekRo — COMMENTED** [review]
Please comment one line of code - seems useless.

File: index.html (hunk 1 above)
Comment (KamilaKisiel) [reply index.html]: sloppy me. gonna fix it asap

Comment (RadekRo) [reply index.html]: ok Push it on the same branch - your pull request should be correct than and I can merge it to develop ;)

**Review by RadekRo — COMMENTED** [review]
ok. Let me know on slack when you finish this. Thanks.

**Comment by KamilaKisiel** [comment]
yo! job done

**Review by RadekRo — APPROVED** [review]
Everything looks ok to me now. Agreed to merge.

**Review by JarekSopinski — APPROVED** [review]

**Pull request merged**